src/Share.jsx

```diff
@@ -125,12 +133,7 @@ export function getModelPath(installPrefix, pathPrefix, urlParams) {
   if (filepath === '') {
     return null
   }
-  const splitRegex = /\.ifc/i
-  const match = splitRegex.exec(filepath)
-  if (!match) {
-    throw new Error('Filepath must contain ".ifc" (case-insensitive)')
-  }
-  const parts = filepath.split(splitRegex)
+  const {parts, match} = splitAroundExtension(filepath)
```

Comment:
`splitAroundExtension` raises an exception if an unsupported file path is passed to it, which is not caught here?

Does it make sense to perform a check with `isExtensionSupported` prior to performing the split, or handling the exception, or adding a `TODO` for the future?

Reply:
Good catch.  I restored use of `alert` and bubble it up.

Done.